pkg/model/request_entity_test.go

```diff
@@ -49,13 +49,13 @@ func testRequestEntity(t *testing.T, describe spec.G, it spec.S) {
 			assert.Equal(t, envFake, rawSubject.env)
 		})
 
-		it("sets the buffer stock", func() {
-			assert.Equal(t, bufferStock, rawSubject.bufferStock)
+		it("sets the routing stock", func() {
+			assert.Equal(t, routingStock, rawSubject.routingStock)
 		})
 
-		it("sets the retry backoff duration to 100ms", func() {
-			assert.Equal(t, 100*time.Millisecond, rawSubject.nextBackoff)
-		})
+		//it("sets the retry backoff duration to 100ms", func() {
```

Comment:
Delete.

Reply:
Deleted it in https://github.com/pivotal/skenario/pull/67/commits/b3c3d8657d2149597dd78d8f0701746ca53862f2 commit.